dotCMS/src/test/java/com/dotcms/cube/CubeJSClientTest.java

```diff
@@ -136,7 +137,16 @@ public void http404() {
             final CubeJSClient cubeClient = new CubeJSClient(
                     String.format("http://%s:%s", cubeServerIp, cubeJsServerPort),
                     getAccessToken());
-            final CubeJSResultSet cubeJSResultSet = cubeClient.send(cubeJSQuery);
+            CubeJSResultSet cubeJSResultSet = new CubeJSResultSetImpl(List.of());
```

Comment:
this line is not ok

it should keep as before I mean 


final CubeJSResultSet cubeJSResultSet = cubeClient.send(cubeJSQuery);

the change that broke this test was 

[https://github.com/dotCMS/core/pull/26104/files#diff-6710105ea300c1ca8d936cd53f76b7ad3e29073acbe272457758dd72b1f63f13R311

and

](https://github.com/dotCMS/core/pull/26104/files#diff-21e76b1df7d7df5fc76a2bab441929ea7c7808dd665f8265eea49548a887b240R120)

Reply:
@freddyucv @victoralfaro-dotcms is going to fix this test